frontend/assets/tailwind.css

```diff
@@ -0,0 +1,5 @@
+@tailwind base;
+
+@tailwind components;
+
```

Comment:
remove this line.

Reply:
Removint @tailwind base destroys nav tailwind component.

Reply:
Oh you are talking about empty line between them?

Reply:
Łukasz misunderstood , we wil remove the empty line, thanks!

Reply:
i am not see change

Reply:
They weren't pushed to origin, now they should be visible.